client/js/admin.js

```diff
@@ -27,11 +27,11 @@ const submit = function() {
 
 	$("#status").text("Uploading...")
 
-	const XHR = new XMLHttpRequest()
-	XHR.open("POST", "")
-	XHR.setRequestHeader("Content-Type", "application/json")
+	const xhr = new XMLHttpRequest()
```

Comment:
so i get why you changed this, and that's totally fine, but for some reason most places make XHR capitalized because its an acronym /shrug